.github/CODEOWNERS

```diff
@@ -224,6 +224,7 @@ packages/react-components/react-breadcrumb @microsoft/cxe-prg
 packages/react-components/react-drawer @microsoft/cxe-prg
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 765 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27379&commitId=f38e0ed91bd9f9db205b463955f44b77500374ac&env=prod)

✅ There was 0 screenshots added, 6 screenshots removed, 1138 screenshots unchanged, 0 screenshots with different dimensions and 759 screenshots with visible difference.


  <details><summary><b>unknown</b> 765 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|232| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|499| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|278| Changed|
|Avatar Converged.basic.normal.chromium.png|266| Changed|
|Avatar Converged.color - Dark Mode.normal.chromium.png|1934| Changed|
|Avatar Converged.color - High Contrast.normal.chromium.png|3425| Changed|
|Avatar Converged.color+active - Dark Mode.normal.chromium.png|1934| Changed|
|Avatar Converged.color+active - High Contrast.normal.chromium.png|3425| Changed|
|Avatar Converged.color+active.normal.chromium.png|1956| Changed|
|Avatar Converged.color.normal.chromium.png|1956| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|1205| Changed|
|Avatar Converged.image-bad-url.normal.chromium.png|55| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|819| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|1795| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|611| Changed|
|Avatar Converged.size+name.normal.chromium.png|1398| Changed|
|AvatarGroup Converged.basic - Dark Mode.default.chromium.png|6438| Changed|
|AvatarGroup Converged.basic - High Contrast.default.chromium.png|11224| Changed|
|AvatarGroup Converged.basic.default.chromium.png|6463| Changed|
|AvatarGroup Converged.layoutPie - Dark Mode.default.chromium.png|2079| Changed|
|AvatarGroup Converged.layoutPie - High Contrast.default.chromium.png|3643| Changed|
|AvatarGroup Converged.layoutPie - RTL.default.chromium.png|2028| Changed|
|AvatarGroup Converged.layoutPie-1 - Dark Mode.default.chromium.png|64| Changed|
|AvatarGroup Converged.layoutPie-1 - High Contrast.default.chromium.png|115| Changed|
|AvatarGroup Converged.layoutPie-1 - RTL.default.chromium.png|60| Changed|
|AvatarGroup Converged.layoutPie-1.default.chromium.png|64| Changed|
|AvatarGroup Converged.layoutPie-2 - Dark Mode.default.chromium.png|100| Changed|
|AvatarGroup Converged.layoutPie-2 - High Contrast.default.chromium.png|172| Changed|
|AvatarGroup Converged.layoutPie-2 - RTL.default.chromium.png|101| Changed|
|AvatarGroup Converged.layoutPie-2.default.chromium.png|101| Changed|
|AvatarGroup Converged.layoutPie.default.chromium.png|2088| Changed|
|AvatarGroup Converged.layoutStack - Dark Mode.default.chromium.png|6409| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|11174| Changed|
|AvatarGroup Converged.layoutStack.default.chromium.png|6434| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.chromium.png|287| Changed|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|539| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.chromium.png|501| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|955| Changed|
|AvatarGroup Converged.overflowContent.chromium.png|288| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|543| Changed|
|AvatarGroup Converged.overflowIndicator - Dark Mode.default.chromium.png|6438| Changed|
|AvatarGroup Converged.overflowIndicator - High Contrast.default.chromium.png|11224| Changed|
|AvatarGroup Converged.overflowIndicator.default.chromium.png|6463| Changed|
|Button Converged.As an anchor.default.chromium.png|339| Changed|
|Button Converged.As an anchor.hover.chromium.png|346| Changed|
|Button Converged.As an anchor.pressed.chromium.png|565| Changed|
|Button Converged.Circular and icon only - RTL.default.chromium.png|147| Changed|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|339| Changed|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|541| Changed|
|Button Converged.Circular and icon only.default.chromium.png|147| Changed|
|Button Converged.Circular and icon only.hover.chromium.png|339| Changed|
|Button Converged.Circular and icon only.pressed.chromium.png|541| Changed|
|Button Converged.Circular.default.chromium.png|334| Changed|
|Button Converged.Circular.hover.chromium.png|341| Changed|
|Button Converged.Circular.pressed.chromium.png|537| Changed|
|Button Converged.Default - Dark Mode.default.chromium.png|19332| Changed|
|Button Converged.Default - Dark Mode.hover.chromium.png|19287| Changed|
|Button Converged.Default - Dark Mode.pressed.chromium.png|19386| Changed|
|Button Converged.Default - High Contrast.default.chromium.png|23135| Changed|
|Button Converged.Default - High Contrast.hover.chromium.png|22634| Changed|
|Button Converged.Default - High Contrast.pressed.chromium.png|22634| Changed|
|Button Converged.Default - RTL.default.chromium.png|340| Changed|
|Button Converged.Default - RTL.hover.chromium.png|347| Changed|
|Button Converged.Default - RTL.pressed.chromium.png|566| Changed|
|Button Converged.Default.default.chromium.png|339| Changed|
|Button Converged.Default.hover.chromium.png|346| Changed|
|Button Converged.Default.pressed.chromium.png|565| Changed|
|Button Converged.Disabled - Dark Mode.default.chromium.png|19323| Changed|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|19323| Changed|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|19323| Changed|
|Button Converged.Disabled - High Contrast.default.chromium.png|23006| Changed|
|Button Converged.Disabled - High Contrast.hover.chromium.png|23006| Changed|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|23006| Changed|
|Button Converged.Disabled.default.chromium.png|214| Changed|
|Button Converged.Disabled.hover.chromium.png|214| Changed|
|Button Converged.Disabled.pressed.chromium.png|214| Changed|
|Button Converged.Icon only.default.chromium.png|144| Changed|
|Button Converged.Icon only.hover.chromium.png|336| Changed|
|Button Converged.Icon only.pressed.chromium.png|540| Changed|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|19158| Changed|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|19158| Changed|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|19158| Changed|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|23006| Changed|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|23006| Changed|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|23006| Changed|
|Button Converged.Outline Disabled.default.chromium.png|125| Changed|
|Button Converged.Outline Disabled.hover.chromium.png|125| Changed|
|Button Converged.Outline Disabled.pressed.chromium.png|125| Changed|
|Button Converged.Outline.default.chromium.png|339| Changed|
|Button Converged.Outline.hover.chromium.png|352| Changed|
|Button Converged.Outline.pressed.chromium.png|381| Changed|
|Button Converged.Primary - Dark Mode.default.chromium.png|19227| Changed|
|Button Converged.Primary - Dark Mode.hover.chromium.png|19205| Changed|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|19292| Changed|
|Button Converged.Primary - High Contrast.default.chromium.png|22574| Changed|
|Button Converged.Primary - High Contrast.hover.chromium.png|22632| Changed|
|Button Converged.Primary - High Contrast.pressed.chromium.png|22632| Changed|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|19326| Changed|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|19326| Changed|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|19326| Changed|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|22923| Changed|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|22923| Changed|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|22923| Changed|
|Button Converged.Primary Disabled.default.chromium.png|193| Changed|
|Button Converged.Primary Disabled.hover.chromium.png|193| Changed|
|Button Converged.Primary Disabled.pressed.chromium.png|193| Changed|
|Button Converged.Primary.default.chromium.png|1454| Changed|
|Button Converged.Primary.hover.chromium.png|1547| Changed|
|Button Converged.Primary.pressed.chromium.png|1810| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|1891| Changed|
|Button Converged.Size large.default.chromium.png|519| Changed|
|Button Converged.Size large.hover.chromium.png|738| Changed|
|Button Converged.Size large.pressed.chromium.png|935| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|1497| Changed|
|Button Converged.Size small - with long text wrapping.chromium.png|853| Changed|
|Button Converged.Size small.default.chromium.png|247| Changed|
|Button Converged.Size small.hover.chromium.png|415| Changed|
|Button Converged.Size small.pressed.chromium.png|484| Changed|
|Button Converged.Subtle - Dark Mode.default.chromium.png|19268| Changed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|19276| Changed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|19282| Changed|
|Button Converged.Subtle - High Contrast.default.chromium.png|23009| Changed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|22632| Changed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|22632| Changed|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|19148| Changed|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|19148| Changed|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|19148| Changed|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|22923| Changed|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|22923| Changed|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|22923| Changed|
|Button Converged.Subtle Disabled.default.chromium.png|83| Changed|
|Button Converged.Subtle Disabled.hover.chromium.png|83| Changed|
|Button Converged.Subtle Disabled.pressed.chromium.png|83| Changed|
|Button Converged.Subtle.default.chromium.png|240| Changed|
|Button Converged.Subtle.hover.chromium.png|275| Changed|
|Button Converged.Subtle.pressed.chromium.png|506| Changed|
|Button Converged.Transparent - Dark Mode.default.chromium.png|19268| Changed|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|19197| Changed|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|19172| Changed|
|Button Converged.Transparent - High Contrast.default.chromium.png|23009| Changed|
|Button Converged.Transparent - High Contrast.hover.chromium.png|22632| Changed|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|22632| Changed|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|19148| Changed|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|19148| Changed|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|19148| Changed|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|22923| Changed|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|22923| Changed|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|22923| Changed|
|Button Converged.Transparent Disabled.default.chromium.png|83| Changed|
|Button Converged.Transparent Disabled.hover.chromium.png|83| Changed|
|Button Converged.Transparent Disabled.pressed.chromium.png|83| Changed|
|Button Converged.Transparent.default.chromium.png|240| Changed|
|Button Converged.Transparent.hover.chromium.png|210| Changed|
|Button Converged.Transparent.pressed.chromium.png|223| Changed|
|Button Converged.With icon after content - RTL.default.chromium.png|389| Changed|
|Button Converged.With icon after content - RTL.hover.chromium.png|532| Changed|
|Button Converged.With icon after content - RTL.pressed.chromium.png|605| Changed|
|Button Converged.With icon after content.default.chromium.png|392| Changed|
|Button Converged.With icon after content.hover.chromium.png|543| Changed|
|Button Converged.With icon after content.pressed.chromium.png|615| Changed|
|Button Converged.With icon before content - RTL.default.chromium.png|367| Changed|
|Button Converged.With icon before content - RTL.hover.chromium.png|488| Changed|
|Button Converged.With icon before content - RTL.pressed.chromium.png|566| Changed|
|Button Converged.With icon before content.default.chromium.png|374| Changed|
|Button Converged.With icon before content.hover.chromium.png|488| Changed|
|Button Converged.With icon before content.pressed.chromium.png|567| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|347| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|347| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|347| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|352| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|462| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|351| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|347| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|347| Changed|
|Card Converged.CardHeader.normal.chromium.png|146| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|1407| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|1847| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|1406| Changed|
|Card Converged.appearance.normal.chromium.png|1388| Changed|
|Card Converged.orientation.chromium.png|388| Changed|
|Card Converged.size.normal.chromium.png|1041| Changed|
|CompoundButton Converged.Circular and icon only - RTL.default.chromium.png|484| Changed|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|1051| Changed|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|1383| Changed|
|CompoundButton Converged.Circular and icon only.default.chromium.png|484| Changed|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|1051| Changed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|1383| Changed|
|CompoundButton Converged.Circular.default.chromium.png|808| Changed|
|CompoundButton Converged.Circular.hover.chromium.png|865| Changed|
|CompoundButton Converged.Circular.pressed.chromium.png|1183| Changed|
|CompoundButton Converged.Default - RTL.default.chromium.png|768| Changed|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|1152| Changed|
|CompoundButton Converged.Default.default.chromium.png|784| Changed|
|CompoundButton Converged.Default.pressed.chromium.png|1167| Changed|
|CompoundButton Converged.Disabled.default.chromium.png|403| Changed|
|CompoundButton Converged.Disabled.hover.chromium.png|403| Changed|
|CompoundButton Converged.Disabled.pressed.chromium.png|403| Changed|
|CompoundButton Converged.Icon only.default.chromium.png|478| Changed|
|CompoundButton Converged.Icon only.hover.chromium.png|1037| Changed|
|CompoundButton Converged.Icon only.pressed.chromium.png|1415| Changed|
|CompoundButton Converged.Outline Disabled.default.chromium.png|273| Changed|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|273| Changed|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|273| Changed|
|CompoundButton Converged.Outline.default.chromium.png|784| Changed|
|CompoundButton Converged.Outline.hover.chromium.png|871| Changed|
|CompoundButton Converged.Outline.pressed.chromium.png|892| Changed|
|CompoundButton Converged.Primary Disabled.default.chromium.png|388| Changed|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|388| Changed|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|388| Changed|
|CompoundButton Converged.Primary.default.chromium.png|2800| Changed|
|CompoundButton Converged.Primary.hover.chromium.png|2979| Changed|
|CompoundButton Converged.Primary.pressed.chromium.png|3486| Changed|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|2921| Changed|
|CompoundButton Converged.Size large.default.chromium.png|1077| Changed|
|CompoundButton Converged.Size large.hover.chromium.png|1140| Changed|
|CompoundButton Converged.Size large.pressed.chromium.png|1139| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|2341| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|1916| Changed|
|CompoundButton Converged.Size small.default.chromium.png|717| Changed|
|CompoundButton Converged.Size small.hover.chromium.png|769| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|776| Changed|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|243| Changed|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|243| Changed|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|243| Changed|
|CompoundButton Converged.Subtle.default.chromium.png|708| Changed|
|CompoundButton Converged.Subtle.hover.chromium.png|788| Changed|
|CompoundButton Converged.Subtle.pressed.chromium.png|1128| Changed|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|243| Changed|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|243| Changed|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|243| Changed|
|CompoundButton Converged.Transparent.default.chromium.png|708| Changed|
|CompoundButton Converged.Transparent.hover.chromium.png|620| Changed|
|CompoundButton Converged.Transparent.pressed.chromium.png|660| Changed|
|CompoundButton Converged.With icon after content - RTL.default.chromium.png|1088| Changed|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|1367| Changed|
|CompoundButton Converged.With icon after content.default.chromium.png|1087| Changed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|1367| Changed|
|CompoundButton Converged.With icon before content - RTL.default.chromium.png|786| Changed|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|850| Changed|
|CompoundButton Converged.With icon before content.default.chromium.png|803| Changed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|866| Changed|
|Dialog.Fluid Actions End.chromium.png|16490| Changed|
|Dialog.Fluid Actions Start.chromium.png|17019| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|24873| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|34719| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|15892| Changed|
|Dialog.actions position start & position end.chromium.png|15906| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|24504| Changed|
|Dialog.actions position start - High Contrast.chromium.png|33999| Changed|
|Dialog.actions position start - RTL.chromium.png|15539| Changed|
|Dialog.actions position start.chromium.png|15558| Changed|
|Dialog.alert - Dark Mode.chromium.png|24496| Changed|
|Dialog.alert - High Contrast.chromium.png|33948| Changed|
|Dialog.alert - RTL.chromium.png|15533| Changed|
|Dialog.alert.chromium.png|15547| Changed|
|Dialog.default - Dark Mode.chromium.png|24496| Changed|
|Dialog.default - High Contrast.chromium.png|33948| Changed|
|Dialog.default - RTL.chromium.png|15533| Changed|
|Dialog.default.chromium.png|15547| Changed|
|Dialog.nested - Dark Mode.chromium.png|19249| Changed|
|Dialog.nested - High Contrast.chromium.png|27008| Changed|
|Dialog.nested - RTL.chromium.png|10618| Changed|
|Dialog.nested.chromium.png|10616| Changed|
|Dialog.no actions - Dark Mode.chromium.png|11560| Changed|
|Dialog.no actions - High Contrast.chromium.png|13876| Changed|
|Dialog.no actions - RTL.chromium.png|218| Changed|
|Dialog.no actions.chromium.png|218| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|11560| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|13876| Changed|
|Dialog.no title & no actions - RTL.chromium.png|218| Changed|
|Dialog.no title & no actions.chromium.png|218| Changed|
|Dialog.no title - Dark Mode.chromium.png|23631| Changed|
|Dialog.no title - High Contrast.chromium.png|32914| Changed|
|Dialog.no title - RTL.chromium.png|14644| Changed|
|Dialog.no title.chromium.png|14658| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|36683| Changed|
|Dialog.non-modal - High Contrast.chromium.png|46760| Changed|
|Dialog.non-modal - RTL.chromium.png|11436| Changed|
|Dialog.non-modal.chromium.png|11451| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|10032| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|13565| Changed|
|Dialog.scroll long content - RTL.chromium.png|3057| Changed|
|Dialog.scroll long content.chromium.png|3080| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|11683| Changed|
|Dialog.title custom action - High Contrast.chromium.png|14059| Changed|
|Dialog.title custom action - RTL.chromium.png|354| Changed|
|Dialog.title custom action.chromium.png|354| Changed|
|Dropdown Converged.Disabled with value.default.chromium.png|18| Changed|
|Dropdown Converged.With multiselect value.default.chromium.png|180| Changed|
|Dropdown Converged.With placeholder.default.chromium.png|140| Changed|
|Dropdown Converged.With value.default.chromium.png|130| Changed|
|Field.Checkbox-error.default.chromium.png|135| Changed|
|Field.Combobox-error.default.chromium.png|135| Changed|
|Field.Dropdown-error.default.chromium.png|135| Changed|
|Field.ProgressBar-error.default.chromium.png|135| Changed|
|Field.RadioGroup-error.default.chromium.png|135| Changed|
|Field.Select-error.default.chromium.png|135| Changed|
|Field.Slider-error.default.chromium.png|135| Changed|
|Field.SpinButton-error.default.chromium.png|135| Changed|
|Field.Switch-error.default.chromium.png|135| Changed|
|Field.Textarea-error.default.chromium.png|135| Changed|
|Field.horizontal+longLabel.default.chromium.png|166| Changed|
|Field.validation-error.default.chromium.png|135| Changed|
|Field.validation-none.default.chromium.png|6785| Changed|
|Field.validation-success.default.chromium.png|20094| Changed|
|Field.validation-warning.default.chromium.png|166| Changed|
|Field.validationMessageIcon.default.chromium.png|276| Changed|
|FluentProvider CustomStyleHooks.Button.chromium.png|1744| Changed|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|2065| Changed|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|1573| Changed|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|1306| Changed|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|1744| Changed|
|InfoButton.default - Dark Mode.focus.chromium.png|193| Changed|
|InfoButton.default - Dark Mode.rest.chromium.png|26| Changed|
|InfoButton.default - High Contrast.focus.chromium.png|175| Changed|
|InfoButton.default - High Contrast.rest.chromium.png|39| Changed|
|InfoButton.default.focus.chromium.png|228| Changed|
|InfoButton.default.rest.chromium.png|29| Changed|
|InfoLabel.default - Dark Mode.rest.chromium.png|26| Changed|
|InfoLabel.default - High Contrast.rest.chromium.png|39| Changed|
|InfoLabel.default - RTL.rest.chromium.png|29| Changed|
|InfoLabel.default.rest.chromium.png|29| Changed|
|InfoLabel.required - RTL.rest.chromium.png|29| Changed|
|InfoLabel.required.rest.chromium.png|29| Changed|
|InfoLabel.size-large.rest.chromium.png|40| Changed|
|InfoLabel.size-small.rest.chromium.png|22| Changed|
|InfoLabel.wrap.rest.chromium.png|29| Changed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|20| Changed|
|Menu Converged - selection groups.default.selected.chromium.png|20| Changed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|18| Changed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|17| Changed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|20| Changed|
|Menu Converged - selection.checkbox.selected.chromium.png|20| Changed|
|MenuButton Converged.Circular and icon only.default.chromium.png|79| Changed|
|MenuButton Converged.Circular and icon only.hover.chromium.png|147| Changed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|132| Changed|
|MenuButton Converged.Circular.default.chromium.png|344| Changed|
|MenuButton Converged.Circular.hover.chromium.png|341| Changed|
|MenuButton Converged.Circular.pressed.chromium.png|483| Changed|
|MenuButton Converged.Default - RTL.default.chromium.png|340| Changed|
|MenuButton Converged.Default - RTL.hover.chromium.png|335| Changed|
|MenuButton Converged.Default - RTL.pressed.chromium.png|487| Changed|
|MenuButton Converged.Default.default.chromium.png|342| Changed|
|MenuButton Converged.Default.hover.chromium.png|337| Changed|
|MenuButton Converged.Default.pressed.chromium.png|489| Changed|
|MenuButton Converged.Disabled.default.chromium.png|175| Changed|
|MenuButton Converged.Disabled.hover.chromium.png|175| Changed|
|MenuButton Converged.Disabled.pressed.chromium.png|175| Changed|
|MenuButton Converged.Icon only.default.chromium.png|79| Changed|
|MenuButton Converged.Icon only.hover.chromium.png|147| Changed|
|MenuButton Converged.Icon only.pressed.chromium.png|132| Changed|
|MenuButton Converged.Outline Disabled.default.chromium.png|122| Changed|
|MenuButton Converged.Outline Disabled.hover.chromium.png|122| Changed|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|122| Changed|
|MenuButton Converged.Outline.default.chromium.png|342| Changed|
|MenuButton Converged.Outline.hover.chromium.png|353| Changed|
|MenuButton Converged.Outline.pressed.chromium.png|377| Changed|
|MenuButton Converged.Primary Disabled.default.chromium.png|157| Changed|
|MenuButton Converged.Primary Disabled.hover.chromium.png|157| Changed|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|157| Changed|
|MenuButton Converged.Primary.default.chromium.png|1223| Changed|
|MenuButton Converged.Primary.hover.chromium.png|1301| Changed|
|MenuButton Converged.Primary.pressed.chromium.png|1522| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|1898| Changed|
|MenuButton Converged.Size large.default.chromium.png|531| Changed|
|MenuButton Converged.Size large.hover.chromium.png|648| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|950| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|1507| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|855| Changed|
|MenuButton Converged.Size small.default.chromium.png|256| Changed|
|MenuButton Converged.Size small.hover.chromium.png|301| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|427| Changed|
|MenuButton Converged.Subtle Disabled.default.chromium.png|86| Changed|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|86| Changed|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|86| Changed|
|MenuButton Converged.Subtle.default.chromium.png|250| Changed|
|MenuButton Converged.Subtle.hover.chromium.png|277| Changed|
|MenuButton Converged.Subtle.pressed.chromium.png|438| Changed|
|MenuButton Converged.Transparent Disabled.default.chromium.png|86| Changed|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|86| Changed|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|86| Changed|
|MenuButton Converged.Transparent.default.chromium.png|250| Changed|
|MenuButton Converged.Transparent.hover.chromium.png|219| Changed|
|MenuButton Converged.Transparent.pressed.chromium.png|233| Changed|
|MenuButton Converged.With icon.default.chromium.png|400| Changed|
|MenuButton Converged.With icon.hover.chromium.png|544| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|678| Changed|
|Persona Converged.basic - Dark Mode.normal.chromium.png|524| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|844| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|586| Changed|
|Persona Converged.basic.normal.chromium.png|586| Changed|
|Persona Converged.size-avatar.normal.chromium.png|355| Changed|
|Persona Converged.size-presence.normal.chromium.png|2889| Changed|
|Persona Converged.textAlignment.normal.chromium.png|1859| Changed|
|Persona Converged.textPosition.normal.chromium.png|11115| Changed|
|Persona Converged.textWrap.normal.chromium.png|82| Changed|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|395| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|395| Changed|
|PresenceBadge Converged - sizes.default - RTL.chromium.png|740| Changed|
|PresenceBadge Converged - sizes.default.chromium.png|740| Changed|
|PresenceBadge Converged - status.default - RTL.chromium.png|584| Changed|
|PresenceBadge Converged - status.default.chromium.png|583| Changed|
|TabList and Tab Converged.With icon only.default.chromium.png|189| Changed|
|TabList and Tab Converged.With icon only.hover.chromium.png|226| Changed|
|TabList and Tab Converged.With icon only.pressed.chromium.png|233| Changed|
|Table flex - subtle selection.rest.chromium.png|369| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|367| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|624| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|616| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|1167| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|1073| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|718| Changed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|712| Changed|
|Table layout flex - cell actions.always visible.chromium.png|712| Changed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|706| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|311| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|381| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|705| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|726| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|375| Changed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|454| Changed|
|Table layout flex - cell actions.default.chromium.png|369| Changed|
|Table layout flex - cell actions.default.hover row.chromium.png|449| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|369| Changed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|449| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|311| Changed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|311| Changed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|311| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|705| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|705| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|705| Changed|
|Table layout flex - headers.sortable.chromium.png|369| Changed|
|Table layout flex - headers.sortable.hover header.chromium.png|369| Changed|
|Table layout flex - headers.sortable.press header.chromium.png|369| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|369| Changed|
|Table layout flex - truncate.false.chromium.png|369| Changed|
|Table layout flex - truncate.true.chromium.png|369| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|309| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|689| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|370| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|364| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|305| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|677| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|368| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|362| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|310| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|715| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|373| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|367| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|308| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|702| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|372| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|366| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|311| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|705| Changed|
|Table layout flex.multiselect - RTL.chromium.png|375| Changed|
|Table layout flex.multiselect.chromium.png|369| Changed|
|Table layout flex.primary cell.chromium.png|369| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|311| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|705| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|375| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|310| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|689| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|373| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|368| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|309| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|691| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|372| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|367| Changed|
|Table layout flex.single select (checked).chromium.png|369| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|311| Changed|
|Table layout flex.single select - High Contrast.chromium.png|705| Changed|
|Table layout flex.single select - RTL.chromium.png|375| Changed|
|Table layout flex.single select.chromium.png|369| Changed|
|Table layout flex.size - extra small.chromium.png|180| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|311| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|705| Changed|
|Table layout flex.size - medium - RTL.chromium.png|375| Changed|
|Table layout flex.size - medium.chromium.png|369| Changed|
|Table layout flex.size - small.chromium.png|184| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|624| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|616| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|1167| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|1073| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|718| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|712| Changed|
|Table layout table - cell actions.always visible.chromium.png|712| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|706| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|311| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|381| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|705| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|726| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|375| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|454| Changed|
|Table layout table - cell actions.default.chromium.png|369| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|449| Changed|
|Table layout table - cell actions.in header cell.chromium.png|369| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|449| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|311| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|311| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|311| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|705| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|705| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|705| Changed|
|Table layout table - headers.sortable.chromium.png|369| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|369| Changed|
|Table layout table - headers.sortable.press header.chromium.png|369| Changed|
|Table layout table - truncate.default (disabled).chromium.png|369| Changed|
|Table layout table - truncate.false.chromium.png|369| Changed|
|Table layout table - truncate.true.chromium.png|369| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|309| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|689| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|370| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|364| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|305| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|677| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|368| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|362| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|310| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|715| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|373| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|367| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|308| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|702| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|372| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|366| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|311| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|705| Changed|
|Table layout table.multiselect - RTL.chromium.png|375| Changed|
|Table layout table.multiselect.chromium.png|369| Changed|
|Table layout table.primary cell.chromium.png|369| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|311| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|705| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|375| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|310| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|689| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|373| Changed|
|Table layout table.single select (checked) - brand.chromium.png|368| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|309| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|691| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|372| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|367| Changed|
|Table layout table.single select (checked).chromium.png|369| Changed|
|Table layout table.single select - Dark Mode.chromium.png|311| Changed|
|Table layout table.single select - High Contrast.chromium.png|705| Changed|
|Table layout table.single select - RTL.chromium.png|375| Changed|
|Table layout table.single select.chromium.png|369| Changed|
|Table layout table.size - extra small.chromium.png|180| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|311| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|705| Changed|
|Table layout table.size - medium - RTL.chromium.png|375| Changed|
|Table layout table.size - medium.chromium.png|369| Changed|
|Table layout table.size - small.chromium.png|184| Changed|
|Table table - subtle selection.rest.chromium.png|369| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|367| Changed|
|ToggleButton Converged.Checked.default.chromium.png|459| Changed|
|ToggleButton Converged.Checked.hover.chromium.png|346| Changed|
|ToggleButton Converged.Checked.pressed.chromium.png|565| Changed|
|ToggleButton Converged.Circular and icon only.default.chromium.png|147| Changed|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|339| Changed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|541| Changed|
|ToggleButton Converged.Circular.default.chromium.png|334| Changed|
|ToggleButton Converged.Circular.hover.chromium.png|341| Changed|
|ToggleButton Converged.Circular.pressed.chromium.png|537| Changed|
|ToggleButton Converged.Default - Dark Mode.default.chromium.png|19332| Changed|
|ToggleButton Converged.Default - Dark Mode.hover.chromium.png|19287| Changed|
|ToggleButton Converged.Default - Dark Mode.pressed.chromium.png|19386| Changed|
|ToggleButton Converged.Default - High Contrast.default.chromium.png|23135| Changed|
|ToggleButton Converged.Default - High Contrast.hover.chromium.png|22634| Changed|
|ToggleButton Converged.Default - High Contrast.pressed.chromium.png|22634| Changed|
|ToggleButton Converged.Default - RTL.default.chromium.png|340| Changed|
|ToggleButton Converged.Default - RTL.hover.chromium.png|347| Changed|
|ToggleButton Converged.Default - RTL.pressed.chromium.png|566| Changed|
|ToggleButton Converged.Default.default.chromium.png|339| Changed|
|ToggleButton Converged.Default.hover.chromium.png|346| Changed|
|ToggleButton Converged.Default.pressed.chromium.png|565| Changed|
|ToggleButton Converged.Disabled.default.chromium.png|214| Changed|
|ToggleButton Converged.Disabled.hover.chromium.png|214| Changed|
|ToggleButton Converged.Disabled.pressed.chromium.png|214| Changed|
|ToggleButton Converged.Icon only.default.chromium.png|144| Changed|
|ToggleButton Converged.Icon only.hover.chromium.png|336| Changed|
|ToggleButton Converged.Icon only.pressed.chromium.png|540| Changed|
|ToggleButton Converged.Outline.default.chromium.png|339| Changed|
|ToggleButton Converged.Outline.hover.chromium.png|352| Changed|
|ToggleButton Converged.Outline.pressed.chromium.png|381| Changed|
|ToggleButton Converged.Primary - Dark Mode.default.chromium.png|19227| Changed|
|ToggleButton Converged.Primary - Dark Mode.hover.chromium.png|19205| Changed|
|ToggleButton Converged.Primary - Dark Mode.pressed.chromium.png|19292| Changed|
|ToggleButton Converged.Primary - High Contrast.default.chromium.png|22574| Changed|
|ToggleButton Converged.Primary - High Contrast.hover.chromium.png|22632| Changed|
|ToggleButton Converged.Primary - High Contrast.pressed.chromium.png|22632| Changed|
|ToggleButton Converged.Primary Checked.default.chromium.png|1627| Changed|
|ToggleButton Converged.Primary Checked.hover.chromium.png|1547| Changed|
|ToggleButton Converged.Primary Checked.pressed.chromium.png|1810| Changed|
|ToggleButton Converged.Primary Disabled.default.chromium.png|193| Changed|
|ToggleButton Converged.Primary Disabled.hover.chromium.png|193| Changed|
|ToggleButton Converged.Primary Disabled.pressed.chromium.png|193| Changed|
|ToggleButton Converged.Primary.default.chromium.png|1454| Changed|
|ToggleButton Converged.Primary.hover.chromium.png|1547| Changed|
|ToggleButton Converged.Primary.pressed.chromium.png|1810| Changed|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|1891| Changed|
|ToggleButton Converged.Size large.default.chromium.png|519| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|738| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|935| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|1497| Changed|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|853| Changed|
|ToggleButton Converged.Size small.default.chromium.png|247| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|415| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|484| Changed|
|ToggleButton Converged.Subtle Checked.default.chromium.png|425| Changed|
|ToggleButton Converged.Subtle Checked.hover.chromium.png|275| Changed|
|ToggleButton Converged.Subtle Checked.pressed.chromium.png|506| Changed|
|ToggleButton Converged.Subtle Disabled.default.chromium.png|83| Changed|
|ToggleButton Converged.Subtle Disabled.hover.chromium.png|83| Changed|
|ToggleButton Converged.Subtle Disabled.pressed.chromium.png|83| Changed|
|ToggleButton Converged.Subtle.default.chromium.png|240| Changed|
|ToggleButton Converged.Subtle.hover.chromium.png|275| Changed|
|ToggleButton Converged.Subtle.pressed.chromium.png|506| Changed|
|ToggleButton Converged.Transparent Checked.default.chromium.png|210| Changed|
|ToggleButton Converged.Transparent Checked.hover.chromium.png|210| Changed|
|ToggleButton Converged.Transparent Checked.pressed.chromium.png|223| Changed|
|ToggleButton Converged.Transparent Disabled.default.chromium.png|83| Changed|
|ToggleButton Converged.Transparent Disabled.hover.chromium.png|83| Changed|
|ToggleButton Converged.Transparent Disabled.pressed.chromium.png|83| Changed|
|ToggleButton Converged.Transparent.default.chromium.png|240| Changed|
|ToggleButton Converged.Transparent.hover.chromium.png|210| Changed|
|ToggleButton Converged.Transparent.pressed.chromium.png|223| Changed|
|ToggleButton Converged.With icon after content.default.chromium.png|392| Changed|
|ToggleButton Converged.With icon after content.hover.chromium.png|543| Changed|
|ToggleButton Converged.With icon after content.pressed.chromium.png|615| Changed|
|ToggleButton Converged.With icon before content.default.chromium.png|374| Changed|
|ToggleButton Converged.With icon before content.hover.chromium.png|488| Changed|
|ToggleButton Converged.With icon before content.pressed.chromium.png|567| Changed|
|Toolbar Converged.Default.Button Pressed.chromium.png|2108| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|1565| Changed|
|Toolbar Converged.Default.default.chromium.png|1547| Changed|
|Toolbar Converged.Far Group.chromium.png|5881| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|2254| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|1631| Changed|
|Toolbar Converged.Large.default.chromium.png|1612| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|1957| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|1556| Changed|
|Toolbar Converged.Small.default.chromium.png|1542| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|3365| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|3061| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|3048| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|2044| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|1626| Changed|
|Toolbar Converged.Vertical.default.chromium.png|1620| Changed|
|Tooltip Converged.basic - Dark Mode.chromium.png|170| Changed|
|Tooltip Converged.basic - High Contrast.chromium.png|461| Changed|
|Tooltip Converged.basic.chromium.png|246| Changed|
|Tooltip Converged.inverted - Dark Mode.chromium.png|374| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|675| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|3562| Changed|
|Tooltip Converged.inverted.chromium.png|1971| Changed|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|7541| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|2416| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|2417| Changed|
|Tooltip Converged.text-wrapping.chromium.png|804| Changed|
|Tooltip Converged.withArrow - Dark Mode.chromium.png|236| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|621| Changed|
|Tooltip Converged.withArrow.chromium.png|342| Changed|
|InfoButton.default - Dark Mode.active.chromium.png|0| Removed|
|InfoButton.default - Dark Mode.hover.chromium.png|0| Removed|
|InfoButton.default - High Contrast.active.chromium.png|0| Removed|
|InfoButton.default - High Contrast.hover.chromium.png|0| Removed|
|InfoButton.default.active.chromium.png|0| Removed|
|InfoButton.default.hover.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>